packages/manager/modules/exchange/src/account/update/account-update.html

```diff
@@ -220,7 +230,7 @@
                         <!--Display name-->
                         <oui-field
                             class="col-md-6"
-                            label="{{:: 'exchange_account_update_displayName_label' | translate }}"
+                            label="{{:: 'exchange_account_update_displayName_label' | translate }} *"
```

Comment:
Even though this is a usual hint, I'm not sure it is sufficient to indicate field is mandatory. Something to mention that the `*` means mandatory fields could be great to add

Reply:
I have modified. I have standardized with the creation form which indeed puts "(obligatoire)" and not * as the modification form used to.